packages/app/src/components/Dimensions/Dialogs/styles/Dialog.css

```diff
@@ -74,7 +70,14 @@
 .unselected-label {
```

Comment:
It seems that the classes unselected-label and selected-label are no longer in use. Can you double check and remove if that is the case?